lfucg-exactions/server/notes/viewsets.py

```diff
@@ -23,39 +23,62 @@ def get_queryset(self):
 
         child_content_type_string = self.request.query_params.get('content_type', None)
         child_object_id = self.request.query_params.get('object_id', None)
-        parent_content_type_string = self.request.query_params.get('parent_content_type', None)
-        parent_object_id = self.request.query_params.get('parent_object_id', None)
                 
         if child_content_type_string is not None:
             split_child_content = child_content_type_string.split('_')
 
             if len(split_child_content) == 2:
                 child_content_type_app_label = split_child_content[0]
                 child_content_type_model = split_child_content[1]
-
                 child_content_type = ContentType.objects.get(app_label=child_content_type_app_label, model=child_content_type_model)
```

Comment:
this could potential throw an exception if you pass in a parameter string with a typo.